src/generators/css-filter.ts

```diff
@@ -63,7 +79,9 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        // If user has the chrome issue the colors should be the other way around as of the rootcolors will affect the whole background color of the page
+        const rootColors = hasChromiumIssue501582() && config.mode === FilterMode.light ? [0, 0, 0] : [255, 255, 255];
```

Comment:
I created a table to help understand, what value should be.
```
             | dark mode                       | light mode
not Chrome   | [0, 0, 0] remains [0, 0, 0]     | [255, 255, 255] remains [255, 255, 255]
old Chrome   | [0, 0, 0] remains [0, 0, 0]     | [255, 255, 255] remains [255, 255, 255]
issue 501582 | [255, 255, 255] turns [0, 0, 0] | [255, 255, 255] should remain [255, 255, 255]
```

Use `config.mode === FilterMode.dark`. I think it will be the final.

Reply:
Our problem is there are no unit tests. We could just create some test samples and quickly find errors like this. Maybe we will do it later.

Reply:
Later please just say this works and I can Squash those commits